### ISSUE TITLE
Multiple browser support - review alternatives to recaptcha

### ISSUE THREAD

**Issue by richpomfret** [FreeUKGen/FreeBMD2#42]
A user on FreeBMD reported issues with the lack of recaptcha support for the Opera mini browser. While it is not possible to support all browsers, we should investigate and review possible alternatives.

**Comment by PatReynolds** [comment]
recaptcha currently used on postem as a barrier to bots
See 5f6a1862e93790387daff6a9/2957203/

**Comment by PatReynolds** [comment]
@PatReynolds to check the nature of the issue with the team, suggest better pathway, with questions to weed out the wrong'uns, and a simple test (e.g. ask people to spell 10 in English)

**Comment by PatReynolds** [comment]
CAPTCHA are "allowed under the W3C's Web Content Accessibility Guidelines (WCAG) provided that "text alternatives that identify and describe the purpose of the non-text content are provided, and alternative forms of CAPTCHA using output modes for different types of sensory perception are provided to accommodate different disabilities.
It is important to understand the limitation of the WCAG CAPTCHA exemption. It applies only to the content of the CAPTCHA. WCAG still requires that alternative text identify the graphical object as a CAPTCHA. Conformance with all other WCAG guidelines also remains critical for web accessibility."  (https://www.w3.org/TR/turingtest/, checked that it is the current version 21/10/2020).

Potential solutions are a (better) spam filter, honeypot, recaptcha v3 or a logic test (or a combination of these). Support@emailed 21/10/2020

**Comment by PatReynolds** [comment]
@PatReynolds to see if recaptcha v3 is working on Opera

**Comment by PatReynolds** [comment]
Could not find a site where I could test, but https://support.google.com/recaptcha/answer/6223828?hl=en does not include Opera in its list of supported browsers.  NB only two latest versions of those it _does_ support are supported.  We know a major user is stuck on an earlly versino of IE, so don't think recaptcha v3 is a solution.

Suggest implementing a logic test and see if this is sufficient.

**Comment by PatReynolds** [comment]
@PatReynolds  to check "I am not a robot" traps as a solution

**Comment by PatReynolds** [comment]
Google version uses mouse movements and cookies (tracking history) - so would not work.

Homenypot: https://solutionfactor.net/blog/2014/02/01/honeypot-technique-fast-easy-spam-prevention/ seems ok - @Vino-S  what do you think?

**Comment by richpomfret** [comment]
@Vino-S to take a look and see if this is 'do-able' or not.

**Comment by richpomfret** [comment]
Closing- decided and implemented.
